server/controllers/socket.controller.ts

```diff
@@ -142,6 +142,55 @@ async function handleLeave(userId: string, roomCode: string): Promise<void> {
 	}
 }
 
+/**
+ * Begins voting, clearing previous votes
+ * @param roomCode 4-character code for the room
+ * @returns void
+ */
+async function handleStartVoting(roomCode: string): Promise<void> {
+	const roomData = await lookupRoom(roomCode);
+	if (!roomData) return;
+
+	const voters = roomData.voters.map((voter) => {
```

Comment:
This may just be a personal preference thing, but I typically avoid direct mutation. So instead of mutating selection and confidence for each voter object, I would map to new objects like so:
```ts
const updatedVoters = roomData.voters.map((voter) => ({
   selection: null,
   confidence: null,
});
``` 
IMO this is easier to follow and you don't have to worry about any side effects of mutating the original object.

Reply:
Yup, you're right. Fixed.